content/en/docs/started/installing-kubeflow.md

```diff
@@ -0,0 +1,97 @@
++++
+title = "Installing Kubeflow"
+description = "Overview of installation choices for Kubeflow"
+weight = 20
+
++++
+
+<a id="introduction"></a>
+## Introduction
+
+... short discussion about Kubeflow being a "Kubernetes" platform, with multiple components (not an "app"), 
+... link to kubeflow-overview for more details
+... (TODO) add a link to "Reference"/"Kubeflow Versioning Policies" in overview docs
+
+... short discussion about how Kubeflow is versioned, 
+... that is, components in Kubeflow can maintain their own versions, independent of the project-wide versions (like 1.2, 1.3)
+
+<a id="roll-your-own"></a>
+## Path 1: roll your own
```

Comment:
Order the paths in the best order for a new user. Would you recommend that a new Kubeflow user create their own distribution, or would you recommend that they start with one of the existing distributions? (I would personally recommend that they start with an existing distribution.)

Reply:
I have made this change